lib/ClangImporter/SwiftBridging/swift/bridging

```diff
@@ -158,6 +158,13 @@
 #define SWIFT_NONCOPYABLE \
   __attribute__((swift_attr("~Copyable")))
 
+/// Specifies the ownership semantics (passed as owned/retained or
+/// unowned/unretained) for C++ free/global functions and static methods having
+/// a FRT as return type.
+#define SWIFT_RETURNS_RETAINED __attribute__((swift_attr("returns_retained")))
+#define SWIFT_RETURNS_UNRETAINED                                               \
```

Comment:
Could you please also add a separate doc comment for this macro, so that it shows up in IDEs as a documentation of `SWIFT_RETURNS_UNRETAINED`?